lisa/analysis/tasks.py

```diff
@@ -423,6 +423,46 @@ def df_task_total_residency(self, task):
 
         return residency_df
 
+    @df_task_states.used_events
```

Comment:
Looks like the method is actually using `df_task_total_residency()`, so it should be
```
@TasksAnalysis.df_task_total_residency.used_events
```

Reply:
Mmm... right... I was skipping one derivation... but in case `df_task_total_residency` implementation should change, it's better to follow the chain... will updated.